Update javascript.mdx to include Autocapture configuration and details (#1748)
mixpanel/docs: pages/docs/tracking-methods/sdks/javascript.mdx

// debug is disabled
```

## Autocapture
Starting with SDK version 2.60.0, the Javascript SDK can autocapture events when you enable it. To do so, enable the autocapture configuration:
```javascript
// create an instance of the Mixpanel object
mixpanel.init('YOUR_PROJECT_TOKEN', {
    autocapture: true,    // enable autocapture
});
```

> If you'd like to change the default Autocapture settings, you can use custom configs. These are the default configs:

Reviewer: ```suggestion
If you'd like to change the default Autocapture settings, you can use custom configs. This is the default configuration:
```

```javascript
mixpanel.init('YOUR_PROJECT_TOKEN', {
  autocapture: {
    pageview: "full-url",
    click: true,
    input: true,
    scroll: true,
    submit: true,
    capture_text_content: false,
  },
});
```

**Omitting elements from Autocapture tracking**

> For any element on your website / web application you do not want to track, there are built-in selector options to omit it. You can add any element to the `.mp-no-track` class to omit it. You can also opt an individual element out of being included in any tracking (including the `$elements` prop of another target's event) using `.mp-sensitive`.

Reviewer: ```suggestion
Add the built-in `.mp-no-track` class to any element you wish to exclude from tracking. You can also opt an individual element out of being included in any tracking (including the `$elements` prop of another target's event) using the `.mp-sensitive` class.
```

active voice and a bit more concise
